### ISSUE TITLE
[a11y documentation] Audit Process - How We Audit (Transition from Shared Support to CAIA)

### ISSUE THREAD

**Issue by sara-amanda** [department-of-veterans-affairs/va.gov-team#88884]
# CAIA A11y Support
> [!TIP]
> **DEFINITION OF DONE**: Refer to the task list in this ticket, for actionable items.

## Documentation Links
- 🟢 [**NEW/UPDATED Version**](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/CAIA/accessibility/how-we-audit-2024.md#how-we-audit-for-accessibility) - CAIA Accessibility Specialists
- 🔴 [**OLD Version**](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/shared-support/accessibility/documentation/how-we-audit.md#how-we-audit-for-accessibility) - Shared Support (with Shared Support Contact Info and Processes)

## Description

<details><summary>Toggle to view the description, background and details</summary>

### Background
- **Completed**: 
    - CAIA A11Ys previously went through a documentation audit of all accessibility items across multiple VA teams and platforms. It was identified what was current, needed removed or needed updating. 
    - Two landing pages resulted (Platform and Design sites), which now contain the aggregated information. 
- **Next Steps**: 
    - Take the current information that lives in other GitHub folders (for example) that is no longer being used by a current team, and bring that accessibility information into the CAIA Accessibility GitHub folder and link to that resource. 
    - This aggregation will help all A11Ys and non-A11Ys, consume accessibility-related information. 
    - Credit should be given to the originator and links to the original documentation should be provided. 
    - If the documentation has been updated, it should reflect that note as well. 

### Documentation: "How We Audit"

Currently, we have "How to Audit" linked from [our new accessibility page on the Platform site](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-on-va-gov), to the :file_folder: :red_circle: (**_Former_**) Shared Support Folder.

### Details
_What details are necessary for understanding the specific work or request tracked by this issue?_

#### Important Links
- 🟢 [**NEW/UPDATED Version**](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/CAIA/accessibility/how-we-audit-2024.md#how-we-audit-for-accessibility) - CAIA Accessibility Specialists
- 🔴 [**OLD Version**](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/shared-support/accessibility/documentation/how-we-audit.md#how-we-audit-for-accessibility) - Shared Support (with Shared Support Contact Info and Processes)

#### Notes About the Old Version
The Old Version is currently linked in this section ["Receiving Feedback"](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-on-va-gov#AccessibilityonVA.gov-ReceivingFeedback) of the Accessibility on VA.gov page:
- Navigate to the first bullet: Product Reviews
    - The text "[what CAIA looks for](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/shared-support/accessibility/documentation/how-we-audit.md)" is actually, "what Shared Support looks for" - currently.
    - Quoting/pasting the full bullet point below for quick ref.
    - I can submit a page change request to Platform, once this is reviewed and good to go.
    - The new version is currently flagged as a WIP.

> - Product reviews include existing and potential accessibility issues, and proposed solutions. In general, this is [what CAIA looks for](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/shared-support/accessibility/documentation/how-we-audit.md) and [what tools CAIA uses](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/shared-support/accessibility/documentation/what-accessibility-tools-we-use.md) to audit a product.

</details>

## Menu Selection
_**Work necessary for this ticket to be completed (select all that apply):**_
- [x] Documentation Follow-Up
- [x] [Slack Follow-Up Ticket](https://dsva.slack.com/archives/D04NHP2FGEP/p1721058936855249)

## CAIA Assignee(s)
- **Ticket A11y Lead:**  @sara-amanda
- **Additional A11y Support:** @SarahKay8  
- **OCTO**: @artsymartha68 

## Team Info

<details><summary>Toggle to view team info</summary>

- **Team name:** CAIA
- **OCTO product owner:** Martha Wilkes @artsymartha68 
</details>

## Definition of Done

```[tasklist]
### A11y Tasks
- [x] **Issue:** Content link destination identified as Shared Support
- [x] **GitHub Markdown:** Duplication of existing Shared Support How We Audit .md page (I copied that over as a baseline of information to the CAIA > Accessibility GitHub Folder.)
- [x] **Authoring Credit:** Provided a credit to Shared Support at the bottom of the document (and link to original)
- [x] **Initial Updates**: Shared Support Information replaced with CAIA A11Ys contact information.
- [x] **Additional Updates**: Provided additional information, broke out longer text, so it would be scannable, created a toggle for the detailed testing area information to reduce cognitive overload, and included resource links - to reduce searching for resources (whether it is a VFS Team member, new VA A11Y, A11Y Champion, or a new CAIA A11Y).
```

```[tasklist]
### A11Y/OCTO: Review-Related Tasks
- [x] Internal request to OCTO (Martha) on `7/15/2024`
- [x] SK reviewed for second set of eyes and OK'd it. `7/15/2024`
- [x] Informed CAIA team of WIP in A11Y meeting and provided background/links to review, if time allows. `7/19/2024`
- [x] Martha initial feedback `7/16/2024` reviewing status; `8/12/2024` checking in with Martha via Slack (there is no rush on this item)
- [x] Martha feedback - post review
- [x] Make any necessary updates
- [x] Remove the WIP note
- [x] Request the Platform page to be updated with the new link.
```

```[tasklist]
### Platform-Related Tasks
- [x] Platform feedback on submitted ticket
- [x] Platform completion of link update
```

```[tasklist]
## Ticket Updates (CAIA Internal)
- [x] _Connect to an `Epic` or `Intake` (what body of work is this a part of?)_
- [x] _Label with `Originator/Team` (product team or stakeholder requesting support)_
- [x] _Label date in the `Open Date` field_
- [x] _Label with `Estimate` (level of effort expected for this ticket)_
- [x] _Add `Assignee(s)` name(s) to ticket_
- [x] _Add `Assignee(s) name(s) to each task` they will complete via handle tag (if known)_
- [x] _Select a `Priority Level`_
- [x] _Update date in `Last Checked` field_
- [x] _Label with `Actual` (level of effort it took to complete this ticket)_
- [x] _Update date in_ `Closed  Date`
```

**Comment by sara-amanda** [comment]
@artsymartha68 how would you like to proceed?

**Comment by artsymartha68** [comment]
Wondering if a question should be added: "How is a CAIA audit different or the same as a VA 508 Office audit?"

**Comment by coforma-jamie** [comment]
@artsymartha68 @sara-amanda I added this to the doc:

> ### What's the difference between an audit from CAIA, accessibility feedback from the Governance team, and Section 508 testing?
> - CAIA and the Governance team audit products to ensure **usability** by people of all abilities. The focus is on the overall experience, not legal compliance.
>    - Contact CAIA at the start of the Collaboration Cycle. We'll review your product before your major touchpoints with the Governance team.
>    - Governance will review your product at Midpoint and Staging Review. They'll provide accessibility feedback and create defect tickets for you to correct issues. 
> - The Section 508 office audits products for **compliance** with Section 508 standards and policies. The focus is legal compliance. If anything doesn't meet compliance standards, they'll send you a report of issues to fix.

Does this work?

**Comment by coforma-jamie** [comment]
This page has been written - ready for closure review @sara-amanda

**Comment by sara-amanda** [comment]
> [!NOTE]
> ## CAIA INTERNAL
> - [x] This work has been completed
> - [x] Closing ticket
> - [x] Adding to tracker for CAIA Sprint 140
